docs/Overview/FrozenSoilText.md

```diff
@@ -1,8 +1,8 @@
 # Frozen Soil Model
 
-When running in cold regions, the effects of frozen soil on the hydrologic process can be very important. Ice in the soil reduces the amount of precipitation and snow melt that can infiltrate the surface, a high enough ice content can make the soil nearly impermiable. Frozen soil also stores more soil moisture through the winter. Frozen soil moisture cannot drain, or be evaporated, so spring soil moisture contents will be significantly higher in regions with frozen soil, than in regions without them.
+When running in cold regions, the effects of frozen soil on the hydrologic process can be very important. Ice in the soil reduces the amount of precipitation and snow melt that can infiltrate the surface, a high enough ice content can make the soil nearly impermeable. Frozen soil also stores more soil moisture through the winter. Frozen soil moisture cannot drain, or be evaporated, so spring soil moisture contents will be significantly higher in regions with frozen soil, than in regions without them.
```

Comment:
"When running in cold regions, the effects of frozen soil on the hydrologic process can be very important."
This is also true if you are driving or walking in cold regions. I suggest changing this to
"Frozen soils can affect hydrological processes."